core/src/com/unciv/logic/github/Github.kt

```diff
@@ -365,26 +362,29 @@ object Github {
     /** Get a Pixmap from a "preview" png or jpg file at the root of the repo, falling back to the
      *  repo owner's avatar [avatarUrl]. The file content url is constructed from [modUrl] and [defaultBranch]
      *  by replacing the host with `raw.githubusercontent.com`.
+     *  
+     *  @return [Pixmap] on success or `null` on failure
      */
-    fun tryGetPreviewImage(modUrl: String, defaultBranch: String, avatarUrl: String?): Pixmap? {
+    suspend fun getPreviewImageOrNull(modUrl: String, defaultBranch: String, avatarUrl: String?): Pixmap? {
```

Comment:
it works and I did test it. also, naming try on everything feels weird. specially for a method that does not throw any exception.